lib/datasource/bitbucket-tags/index.ts

```diff
@@ -56,17 +60,18 @@ export class BitBucketTagsDatasource extends Datasource {
       await this.bitbucketHttp.getJson<utils.PagedResult<BitbucketTag>>(url)
     ).body;
 
-    const dependency: ReleaseResult = {
-      sourceUrl: BitBucketTagsDatasource.getSourceUrl(repo, registryUrl),
-      registryUrl: BitBucketTagsDatasource.getRegistryURL(registryUrl),
-      releases: null,
-    };
-    dependency.releases = bitbucketTags.values.map(({ name, target }) => ({
+    const releases = bitbucketTags.values.map(({ name, target }) => ({
       version: name,
       gitRef: name,
       releaseTimestamp: target?.date,
     }));
 
+    const dependency: ReleaseResult = {
+      sourceUrl: BitBucketTagsDatasource.getSourceUrl(repo, registryUrl),
+      registryUrl: BitBucketTagsDatasource.getRegistryURL(registryUrl),
```

Comment:
I think we should remove `registryUrl` as it's not really used. We can also disable `customRegistrySupport`.
```suggestion
```